modules/catalog/src/main/java/org/apache/ignite/internal/catalog/CatalogManager.java

```diff
@@ -44,7 +44,7 @@ public interface CatalogManager extends IgniteComponent, CatalogService {
      * @param commands Commands to execute.
      * @return Future representing result of execution.
      */
-    CompletableFuture<Void> execute(List<CatalogCommand> commands);
+    CompletableFuture<Void> execute(Collection<CatalogCommand> commands);
```

Comment:
It is better not to change this signature, because the order of commands matters.

Reply:
Ok, fixed.